contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,336 @@
+
+| **Pros**                                                                                                                 | **Cons**                                                                                                                 |
+|--------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------| 
+| <span className="text-green text-lg">✔</span> It's a cumulative metric that's influenced by all parts of your startup    | <span className="text-red text-lg">✖</span> It's a significantly lagging indicator                                       |
+| <span className="text-green text-lg">✔</span> It's easy to calculate – it's just two numbers any founder should have     | <span className="text-red text-lg">✖</span> It's not useful if you're pre-revenue                                        |
+| <span className="text-green text-lg">✔</span> A steady decline is a strong indicator of product-market fit               | <span className="text-red text-lg">✖</span> It won't help you understand why you do or don't have product-market fit     |
+| <span className="text-green text-lg">✔</span> It's useful for investors                                                  | <span className="text-red text-lg">✖</span> Skewed by non-product factors – e.g. poor cost control                       |
+| <span className="text-green text-lg">✔</span> It's easy to understand                                                    |        |
+| <span className="text-green text-lg">✔</span> Useful for mid to late-stage startups                                      |        |
+
+## Indicator #6 CLV/CAC ratio
+
+> **CLV** = customer lifetime value – i.e. the total value of a customer after churn and the cost of supporting / retaining them.
+>
+> **CAC** = customer acquisition cost - i.e. total sales and marketing spend divided by how many customers you acquired.
+
+CLV/CAC ratio is a robust but complex measure of product-market fit. Robust because it speaks directly to the long-term financial health of your company; complex because calculating CLV accurately isn't a trivial task.
+
+Customer lifetime value attempts to quantify how much a customer is worth over their entire lifetime. This means you need to understand:
+
+- Your average deal size.
+- How long customers stay until they churn.
+- Your gross margin – i.e. how much it costs to support and retain customers.
+- How long it'll take for a customer to pay back the cost of acquiring them. 
+
+Consequently, the ratio is best used by companies selling to enterprises, where sales costs are higher and play a larger role in a product's success.  
```

Comment:
How can early stage (pre-product market fit) companies figure out CLV?

Reply:
Again this section needs a bit more on why this matters to product-market fit, seems like a later stage metric.

Reply:
It definitely is a later stage metric, I probably need to make that clearer. This will go in the conclusion, but the TL;DR is validate PMF with "The Trifecta" first, then add the ratio or burn multiple as a test.